releases/v0.8.0.md

```diff
@@ -0,0 +1,38 @@
+# Release Notes for v0.8.0
+
+Release Date: Sep TBD, 2023
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+
+- TODO
+- TODO
+
+## Hardware Support
+
+Confidential Containers is tested with attestation on the following platforms:
+
+- Intel TDX
+- AMD SEV(-ES)
+- Intel SGX
+
+The following platforms are untested or partially supported:
+
+- IBM Secure Execution (SE) on IBM zSystems (s390x) running LinuxONE
+- AMD SEV-SNP
+
+The following platforms are in development:
+
+- ARM CCA (TODO: Find out what is the status of this?)
+
+## Limitations
+
+- TODO
```

Comment:
You should copy the limitations from the previous release and then we will see if there is anything we can remove.